artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/CoreAmqpConverter.java

```diff
@@ -171,15 +165,15 @@ public static AMQPMessage fromCore(ICoreMessage coreMessage, StorageManager stor
             properties.setMessageId("ID:" + message.getInnerMessage().getUserID().toString());
          }
       }
-      Destination destination = message.getJMSDestination();
+      SimpleString destination = message.getJMSDestination();
```

Comment:
I expect get JMSDestination to return a JMSDestination object not a string

Reply:
This is for Conversion only..  there's no JMSDestination.. we would need to depend on artemis-jms-impl that depends on JMS.

Reply:
But then rename the method. Leaving it named JMSxxxxx means you expect a JMS thing